settings.yml

```diff
@@ -0,0 +1,25 @@
+---
+lib_name: lib_i2c
+project: '{{lib_name}}'
+title: '{{lib_name}}: XMOS I2C Library'
```

Comment:
Probably don't need "XMOS" in all our titles

Reply:
done